### PR TITLE
HPO alt text and video transcripts

### PR DESCRIPTION
**Pull request by eadpearce**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

details/summary element for video transcripts:
Screen Shot 2019-09-13 at 14 55 05
Screen Shot 2019-09-13 at 14 55 17

### CODE REVIEW

File: core/sass/pages/invest/_hpo-detail-page.scss [hunk 1]
```diff
@@ -38,10 +36,17 @@
...
+
+    & > *:first-child {
+      margin-top: $spacing-4;
+    }
   }
 
   .proposition-two-grid {
     flex-direction: row;
+    margin-left: -15px;
+    margin-right: -15px;
```
Comment (MichaelWalker) [inline core/sass/pages/invest/_hpo-detail-page.scss]: should this also use the `$spacing-x`?

Comment (eadpearce) [reply core/sass/pages/invest/_hpo-detail-page.scss]: for some reason the sass plugin blows up when i put `-` in front of the variable and i can't figure out how to fix it 🤷‍♀

Comment (MichaelWalker) [reply core/sass/pages/invest/_hpo-detail-page.scss]: oh yeah, I've seen that before. There is a discussion here - https://stackoverflow.com/questions/13296419/sass-negative-variable-value with some options.

(but I won't argue if you think the better solution is the one you've already got)

File: core/static/core/js/dit.components.hideParagraphOnScroll.js [hunk 2]
```diff
@@ -17,9 +18,11 @@ $(document).ready(function() {
         propositionOne.style['margin-top'] = container.offsetHeight + 'px';
         paragraph.style.display = 'none';
         container.style.position = 'fixed';
-      } else if (paragraph.style.display != 'block' && rect.bottom > 30) {
+        container.style.top = '60px';
```
Comment (MichaelWalker) [inline core/static/core/js/dit.components.hideParagraphOnScroll.js]: We could probably do all of this with much less JS if we just added and removed a CSS class to the container when it became stuck and unstuck.

Would also avoid needing to duplicate the 60px value between the JS and CSS.

Comment (eadpearce) [reply core/static/core/js/dit.components.hideParagraphOnScroll.js]: yeah agree. i didn't write this. i would rather have done it with jquery's add/removeClass function lol. but then again, i hate this stupid sticky and would like to remove it altogether. it annoys me more now that we have the sticky header as well...

File: core/sass/pages/invest/_hpo-detail-page.scss (hunk 1 above)
Comment (eadpearce) [reply core/sass/pages/invest/_hpo-detail-page.scss]: oh lol i was sure i tried brackets... i'll have another go

File: core/static/core/js/dit.components.hideParagraphOnScroll.js (hunk 2 above)
Comment (MichaelWalker) [reply core/static/core/js/dit.components.hideParagraphOnScroll.js]: yeah I totally agree!
Maybe we can talk to Michal about just killing it! 😊

File: core/sass/pages/invest/_hpo-detail-page.scss (hunk 1 above)
Comment (eadpearce) [reply core/sass/pages/invest/_hpo-detail-page.scss]: oh hey it worked LOL

**Pull request merged**